### PR TITLE
Sensitivity Profile Experiment documentation v2

### PR DESCRIPTION
**Pull request by skonefal**
Due to recent changes, documentation now has much easier installation instructions.

### CODE REVIEW

**Comment by skonefal** [comment]
run integration tests

**Comment by iwankgb** [comment]
I summon @mbbroberg to the rescue!

File: experiments/memcached-sensitivity-profile/docs/run_experiment.md [hunk 1]
```diff
@@ -19,153 +19,134 @@
...
+ 
+Please paste following snippet into `config.ini` file.
+Some variables should be changed before running the experiment.
+
+1. `REMOTE_SSH_LOGIN` and `REMOTE_SSH_KEY_PATH` should point to user and key that is authorized to SSH on every machine in the experiment cluster.
+1. `MEMCACHED_USER` should be the same as `REMOTE_SSH_LOGIN`
+1. `MEMCACHED_LISTENING_ADDRESS` should be SUT node address.
+1. `EXPERIMENT_PEAK_LOAD` should be provided appropriate to SUT machine. If tiny VM is used, it might be even as low as 100000. If high-end server machine is used, it might be as high as millions requests per second.
+1. `EXPERIMENT_SLO` should be set appropriate to SUT machine (in microseconds). In VM environment, the default value should be increased 10x. 
+1. When SUT node has HyperThreads, `EXPERIMENT_BE_WORKLOADS` should contain `stress-ng-cache-l1` entry to examine Memcached sensitivity to L1 cache interference.
```
Comment (skonefal) [inline experiments/memcached-sensitivity-profile/docs/run_experiment.md]: I don't think I follow. Do we want to prove that running two processes on single cpu-thread makes interference? I'm not sure if this was the goal of L1 aggressor.

File: experiments/memcached-sensitivity-profile/docs/installation.md [hunk 2]
```diff
@@ -17,85 +17,176 @@
...
-Please download binaries from https://github.com/kubernetes/kubernetes/releases, copy hyperkube to directory listed in`$PATH` and run `./hyperkube --make-symlinks` to make it usable by Swan. Hyperkube binary should have executable bit set. Swan expects that Docker will be the default container runtime for Kubernetes. Please make sure that all dependencies for Kubernetes (e.g. Docker, Etcd) are running and in proper versions.
+`wget` must be installed on node.
 
-Binaries should be placed in the directory listed in `$PATH` for `root` user on System Under Test (SUT) and Service machines.
- 
-#### Docker Image (only when experiments are going to be run on Kubernetes)
-User should create the Docker Image with all applications listed below available in `$PATH`. Image should be named `centos-swan-image` and should be based on Centos7 image. It should be available in local Docker images on SUT machine.
+**Docker**
+
+Please install Docker in version 17.03.
```
Comment (skonefal) [inline experiments/memcached-sensitivity-profile/docs/installation.md]: Right. That's user guide, not for image developer, so I'll drop the magic and leave simple instructions from Docker support.

File: experiments/memcached-sensitivity-profile/docs/installation.md [hunk 3]
```diff
@@ -17,85 +17,176 @@
...
+
+Path `/opt/swan/bin` must be included in `$PATH`.
+
+
+Following workloads are installed this way:
+* Memcached 1.4.35 with *thread affinity* patch
+* Stress-ng - *Synthethic stresser that can stress system in various selectable ways*
+* Caffe - *Deep Learning framework to simulate real workload*
+
+Swan also supports [iBench](https://github.com/stanford-mast/iBench) and [Stream Benchmark](https://www.cs.virginia.edu/stream/) workloads that are not deployed by preceding script. Stress-ng can stress system in similar way.
```
Comment (skonefal) [inline experiments/memcached-sensitivity-profile/docs/installation.md]: Link to stress-ng is in Theory section (on the bottom).

File: experiments/memcached-sensitivity-profile/docs/run_experiment.md [hunk 4]
```diff
@@ -19,153 +19,134 @@
...
 ```
 
 Before running `memcached-sensitivity-profile` please ensure that
-* Cassandra is up and running on the cassandra host
-* Snapteld is running on SUT
-* Mutilate binary is copied to proper location on mutilate hosts
-* From SUT node, a privileged user can connect via ssh to Load Generator nodes using keys authorization.
+* Cassandra is up and running on the Services node.
+* Snapteld is running on SUT.
+* Mutilate binary and `cppzmq-devel` library are installed on Service and Load Generator hosts.
```
Comment (skonefal) [inline experiments/memcached-sensitivity-profile/docs/run_experiment.md]: Are you sure that this is `zeromq`, and not .so with headers?

File: experiments/memcached-sensitivity-profile/docs/installation.md (hunk 2 above)
Comment (skonefal) [inline experiments/memcached-sensitivity-profile/docs/installation.md]: Added below.

File: experiments/memcached-sensitivity-profile/docs/run_experiment.md (hunk 1 above)
Comment (iwankgb) [inline experiments/memcached-sensitivity-profile/docs/run_experiment.md]: Cleared offline :)

**Comment by skonefal** [comment]
run integration tests

File: pkg/experiment/sensitivity/topology/isolations.go [hunk 5]
```diff
@@ -25,8 +25,8 @@ import (
 
 var (
 	// For CPU count based isolation policy flags.
-	hpCPUCountFlag = conf.NewIntFlag("experiment_hp_workload_cpu_number", "Number of CPUs assigned to high priority task. CPUs will be assigned autmatically to workloads.", 1)
-	beCPUCountFlag = conf.NewIntFlag("experiment_be_workload_cpu_number", "Number of CPUs assigned to best effort task. CPUs will be assigned autmatically to workloads.", 1)
+	hpCPUCountFlag = conf.NewIntFlag("experiment_hp_workload_cpu_count", "Number of CPUs assigned to high priority task. CPUs will be assigned automatically to workloads.", 1)
```
Comment (skonefal) [inline pkg/experiment/sensitivity/topology/isolations.go]: @mbbroberg which version is better? `experiment_hp_workload_cpu_number` or `experiment_hp_workload_cpu_count`?

**Comment by skonefal** [comment]
@mbbroberg some parts of this doc should be promoted to main Swan documentation in the next week.

**Comment by mbbroberg** [comment]
Great @skonefal - we can work on that on the next iteration.

**Pull request merged**